.gitignore

```diff
@@ -8,6 +8,7 @@ heapdump.*
 core.*
 Snap.*
 /.project
+**/*.class
```

Comment:
Just `*.class` should be sufficient.

Reply:
You're right, thanks. I corrected it in 7dd071979509feaf45c850aa914ebbc6c9b0113f